instrumentation/aws-lambda/aws-lambda-core-1.0/library/README.md

```diff
@@ -7,7 +7,7 @@ This package contains libraries to help instrument AWS lambda functions in your
 To use the instrumentation, configure `OTEL_INSTRUMENTATION_AWS_LAMBDA_HANDLER` env property to your lambda handler method in following format `package.ClassName::methodName`
 and use one of wrappers as your lambda `Handler`.
 
-In order to configure a span flush timeout (default is set to 1 second), please configure `OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT` env property. The value is in seconds.
+In order to configure a span flush timeout (default is set to 10 second), please configure `OTEL_INSTRUMENTATION_AWS_LAMBDA_FLUSH_TIMEOUT` env property. The value is in milliseconds.
```

Comment:
As far as I can tell this is configured in
https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/78b3c6a2855af354bf9ffe0d09ba03adda700938/instrumentation/aws-lambda/aws-lambda-core-1.0/library/src/main/java/io/opentelemetry/instrumentation/awslambdacore/v1_0/internal/WrapperConfiguration.java#L20-L32